lib_eio/mock/eio_mock.mli

```diff
@@ -106,6 +107,9 @@ module Flow : sig
   val on_read : t -> string Handler.actions -> unit
   (** [on_read t actions] configures the values to return from the mock's [read] function. *)
 
+  val on_write : t -> unit Handler.actions -> unit
```

Comment:
I don't think we need this. We can just use the default behaviour of calling `copy`.